boards/shields/nrf2240ek/boards/nrf52833dk_nrf52833.conf

```diff
@@ -1,7 +1,6 @@
 # Options related to the nPM1300 regulator and the workaround
 # against voltage drops on the nRF2240 power supply pin VDDPALDO.
 CONFIG_SENSOR=y
-CONFIG_NPM1300_CHARGER=y
```

Comment:
It should be aligned to new name CONFIG_NPM13XX_CHARGER, I already created PR with fix from recent regression:
https://github.com/nrfconnect/sdk-nrf/pull/23096/files